.github/.goreleaser-release-darwin.yaml

```
@@ -0,0 +1,58 @@
+before:
+  hooks:
+    - go mod tidy
+    - rm -f site/out/bin/coder*
+
+archives:
+  - id: coder-darwin
+    builds: [coder-darwin]
+    format: zip
+
+builds:
+  - id: coder-slim
+    dir: cmd/coder
+    ldflags: ["-s -w -X github.com/coder/coder/buildinfo.tag={{ .Version }}"]
+    env: [CGO_ENABLED=0]
+    goos: [darwin, linux, windows]
+    goarch: [amd64, arm, arm64]
+    goarm: ["7"]
+    # Only build arm 7 for Linux
+    ignore:
+      - goos: windows
+        goarm: "7"
+      - goos: darwin
+        goarm: "7"
+    hooks:
+      # The "trimprefix" appends ".exe" on Windows.
+      post: |
+        cp {{.Path}} site/out/bin/coder-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ trimprefix .Name "coder" }}
+
+  - id: coder-darwin
+    dir: cmd/coder
+    flags: [-tags=embed]
+    ldflags: ["-s -w -X github.com/coder/coder/buildinfo.tag={{ .Version }}"]
+    env: [CGO_ENABLED=0]
+    goos: [darwin]
+    goarch: [amd64, arm64]
+    hooks:
+      # This signs the binary that will be located inside the zip. MacOS
+      # requires the binary to be signed for notarization.
+      post: |
+        sh -c 'codesign -s {{.Env.AC_APPLICATION_IDENTITY}} -f -v --timestamp --options runtime {{.Path}}'
+
+env:
+  # Apple identity for signing!
+  - AC_APPLICATION_IDENTITY=BDB050EB749EDD6A80C6F119BF1382ECA119CCCC
+
+signs:
+  - ids: [coder-darwin]
+    artifacts: archive
+    cmd: ./scripts/sign_darwin.sh
+    args: ["${artifact}"]
+    output: true
+
+release:
+  ids: [coder-darwin]
+
+snapshot:
+  name_template: "{{ .Version }}-devel+{{ .ShortCommit }}"
```

```suggestion
  name_template: "{{ .Version }}+devel.{{ .ShortCommit }}"
```

Maybe the meaning here is different and my assumption wrong, but this change seems more consistent with other uses.